lib/docker/template/cache.rb

```diff
@@ -17,12 +17,10 @@ def context(builder, context)
         builder.repo.cache_dir.rm_rf
         $stderr.puts Simple::Ansi.yellow(format("Copying context for %s", builder.repo))
         cache_dir = builder.repo.cache_dir
-        cache_dir.parent.mkdir_p
+        cache_dir.mkdir_p
 
         readme(builder)
-        context.cp_r(cache_dir.tap(
-          &:rm_rf
-        ))
+        context.join(".").cp_r(cache_dir)
```

Comment:
Once I had created the tag's cache folder earlier, I needed to copy the content of the folder, not the folder itself. Otherwise, instead of having `repos/[reponame]/cache/[tag]/Dockerfile`, I would have `repos/[reponame]/cache/[tag]/[reponame-tagname-random]/Dockerfile.